src/ui/public/visualize/loader/embedded_visualize_handler.ts

```diff
@@ -94,6 +95,9 @@ export class EmbeddedVisualizeHandler {
     });
 
     element.setAttribute(LOADING_ATTRIBUTE, '');
+    if (process.env.NODE_ENV !== 'production') {
```

Comment:
We'll actively be working to get the functional tests against actual builds running in the production environment, so this check should be removed. I also don't see any harm in having it in every version of Kibana. Also, like @timroes pointed out, this will cause issues for testing against cloud as it runs Kibana in production mode.

Reply:
Great, I will remove it